src/component/panels/SpectrumsPanel/SpectraTable.tsx

```diff
@@ -220,6 +245,47 @@ export function SpectraTable(props: SpectraTableProps) {
       enableVirtualScroll
       approxItemHeight={26}
       context={contextMenu}
+      onSortEnd={handleSortEnd}
     />
   );
 }
+
+const ColumnHeader = ({
+  label,
+  col,
+}: {
+  label: string;
+  col: SpectraTableColumn;
+}) => {
+  const dispatch = useDispatch();
+
+  function selectHandler() {
+    const name = (col as PredefinedTableColumn<any>)?.name;
+    let jpath = '';
+    switch (name) {
+      case 'name':
+        jpath = 'display.name';
+        break;
+      case 'solvent':
+        jpath = 'info.solvent';
```

Comment:
Why is this a special case. Should not the rest of the code be adapted so that it is just a regular path ?

Same for display.name.